### PR TITLE
feat: Add app info to settings

### PR DESCRIPTION
**Pull request by holzeis**
Screenshot 2024-01-22 at 12 29 50

### CODE REVIEW

File: webapp/frontend/lib/settings/settings_screen.dart [hunk 1]
```diff
@@ -9,17 +12,62 @@ class SettingsScreen extends StatefulWidget {
...
-          'Settings Screen',
-        ),
-      ],
+    return SizedBox(
+      width: 500,
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          TabBar(
```
Comment (bonomat) [inline webapp/frontend/lib/settings/settings_screen.dart:26]: You love tabs ha? 😅

File: webapp/frontend/lib/settings/settings_service.dart [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+class SettingsService {
+  const SettingsService();
+
+  Future<String> getNodeId() async {
+    // TODO(holzeis): this should come from the config
+    const port = "3001";
+    const host = "localhost";
+
+    try {
+      final response = await http.get(Uri.http('$host:$port', '/api/node'));
```
Comment (bonomat) [inline webapp/frontend/lib/settings/settings_service.dart:12]: I've recently introduced a new HttpClient  which unifies these calls

https://github.com/get10101/10101/blob/dd57e3e6ff645685267929440c50daf9a86b9f40/webapp/frontend/lib/common/http_client.dart#L12-L11

**Review by bonomat — APPROVED** [review]
LGTM

File: webapp/frontend/lib/settings/settings_screen.dart (hunk 1 above)
Comment (holzeis) [reply webapp/frontend/lib/settings/settings_screen.dart:26]: It's fast, we can redesign when we have more time.

**Pull request merged**